lambda/variables.tf

```diff
@@ -41,6 +41,11 @@ variable "lambda_download_files" {
   default     = false
 }
 
+variable "lambda_ecr_scan_notifications" {
+  description = "deploy lambda function to send notifications from ECR scans"
```

Comment:
Capitalize "**L**ambda" for consistency

Reply:
I've changed this one and the two above that were lower case as well.